api/src/main/java/org/pmiops/workbench/api/DataSetController.java

```diff
@@ -187,11 +132,11 @@ public DataSet apply(org.pmiops.workbench.db.model.DataSet dataSet) {
           result.setName(dataSet.getName());
           Iterable<org.pmiops.workbench.db.model.ConceptSet> conceptSets =
               conceptSetDao.findAll(dataSet.getConceptSetId());
-          result.setConceptSets(Streams.stream(conceptSets)
+          result.setConceptSets(StreamSupport.stream(conceptSets.spliterator(), false)
```

Comment:
I couldn't get the streams to work here, it kept giving me import errors, but maybe I was missing something. @NehaBroad when you get in tomorrow, mind taking a look?